pkg/cache/internal/informers_map.go

```diff
@@ -231,6 +235,15 @@ func (ip *specificInformersMap) addInformerToMap(gvk schema.GroupVersionKind, ob
 	return i, ip.started, nil
 }
 
+func (ip *specificInformersMap) findFieldSelectorByGVR(gvr schema.GroupVersionResource) string {
+	gr := schema.GroupResource{Group: gvr.Group, Resource: gvr.Resource}
+	fieldSelector, ok := ip.fieldSelectorByResource[gr]
```

Comment:
Nit: This is an empty string if there isn't a match in the map, so you do not need the `ok` check

Reply:
Done